docs/UserGuide.adoc

```diff
@@ -148,30 +354,118 @@ Format: `exit`
+
+This section describes features that can be added to OTM in version 2.0.
+
+==== Filtering persons: `filter`
+
+Filters a patient/doctor/donor by specific attributes.
+
+Format: `filter [patient/donor/doctor] [options]`
+
+Returns a list of patients/doctors/doctors that has the specified attributes.
+
+Examples:
+
+* `filter t/patient g/M`
+* `filter t/donor g/M n/Chua`
+
+=== Proposed Extensions
+
+This section describes features that can be improved on and its proposed improvements.
+
+==== Mass addition of data
+
+Add more than one patient/doctor/donor from data in a specified text file.
+
+A duplicate is defined as: two persons of the same type having the same ID.
+When a duplicate is detected, it prompts the user to change the data in the text file.
+
+==== Supporting more types of transplants
+
+Currently OTM has patient-donor matching for kidneys only. Future versions can include matching for +
+more types of organs.
+
+==== Processing a match: `process`
+
+Emails the doctor in charge of a patient when an organ match for the patient is found.
+
+Format: `process **NRIC**`
+
+==== Advanced find
+
+Finding in v2.0 is more advanced with the implementation of fuzzy search and partial matching of keywords.
 
-_{explain how the user can enable/disable data encryption}_
-// end::dataencryption[]
 
 == FAQ
 
-*Q*: How do I transfer my data to another Computer? +
-*A*: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous Address Book folder.
+*Q*: How do I transfer my data to another computer? +
+*A*: Install the app in the other computer and overwrite the empty data file it creates with the file +
+that contains the data of your previous OTM folder.
+
+*Q*: What are the steps to find a patient-donor kidney match? +
+*A*: There are three steps to finding a patient-donor match:
+
+1. Do a blood test to determine donor and patient blood type.
+2. Do a tissue typing test to determine the HLA antigens of the patient and the donor.
+3. Do a cross match, which is a trial transfusion.
 
 == Command Summary
 
-* *Add* `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` +
-e.g. `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
-* *Clear* : `clear`
-* *Delete* : `delete INDEX` +
+This section summarises the commands to use this application. The following list
+contains the summary of the commands.
+
+* *Add* `add t/**TYPE** n/**NAME** p/**PHONE** g/**GENDER** ic/**NRIC** d/**DOCTOR**` +
+e.g. `add patient n/John Doe g/M p/high b/A a/29 ic/S9988330G d/1234 /98765432`
+* *Delete* : `delete **INDEX**` +
 e.g. `delete 3`
-* *Edit* : `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]...` +
-e.g. `edit 2 n/James Lee e/jameslee@example.com`
+* *Edit* : `edit ic/**NRIC** [attributes]` +
+e.g. `edit ic/S7896428G n/James Lee `
 * *Find* : `find KEYWORD [MORE_KEYWORDS]` +
 e.g. `find James Jake`
```

Comment:
find t/patient n/Name??

Reply:
I have changed the command format in my latest commit.